README.md

```diff
@@ -30,62 +30,58 @@ This tool provides the following features:
-resourceGroup: foobar-rg # Resource group where the cluster needs to be created
-region: eastus2 # Region name where the cluster needs to be created, should be in smallcase without space 
-headNodeVMSize: STANDARD_D13_V2 # Any Standrad VM Size supported for Head Nodes
-workerNodeVMSize: STANDARD_D13_V2 # Any Standrad VM Size supported for Worker Nodes
+type: SPARK_AND_LLAP # [Optional], values can be either SPARK_ONLY or LLAP_ONLY Default is SPARK_AND_LLAP
+clusterNamePrefix: Foo # This prefix is used while creating the cluster name. Only first three chars are used as prefix from this string
+resourceGroup: <RESOURCE_GROUP> # Resource group where the cluster needs to be created
+region: <REGION> # Region name where the cluster needs to be created, should be in small case without space. Eg: eastus2
+headNodeVMSize: STANDARD_D13_V2 # Any Standard VM Size supported for Head Nodes in HDInsight, https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-supported-node-configuration
+workerNodeVMSize: STANDARD_D13_V2 # Any Standard VM Size supported for Worker Nodes in HDInsight, https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-supported-node-configuration
 workerNodeSize: 3 # Size of worker nodes
-subscription: YOUR_SUBSCRIPTION # Subscription ID
+subscription: <YOUR_SUBSCRIPTION> # Subscription ID
 
 activeDirectory:
   azureEnv: AZURE # Azure Env either AZURE_CHINA/AZURE_GERMANY/AZURE_US_GOVERNMENT , default AZURE
-  clientId: YOUR_CLIENT_ID # Client ID of the service principal
-  tenantId: YOUR_TENANT_ID # Tenant ID of the service principal
-  clientSecret: YOUR_CLIENT_SECRET # Client Secret for the service principal
+  clientId: <YOUR_CLIENT_ID> # Client ID of the service principal
+  tenantId: <YOUR_TENANT_ID> # Tenant ID of the service principal
+  clientSecret: <YOUR_CLIENT_SECRET> # Client Secret for the service principal
 
 clusterCredentials:
-  clusterLoginUsername: YOU_USER_NAME # Ambari username
-  clusterLoginPassword: YOUR_PASSWORD # Ambari password
-  sshUsername: SSH_USER # SSH username
-  sshPassword: SSH_PASSWORD # SSH password
+  clusterLoginUsername: <YOU_USER_NAME> # Ambari username
+  clusterLoginPassword: <YOUR_PASSWORD> # Ambari password
+  sshUsername: <SSH_USER> # SSH username
+  sshPassword: <SSH_PASSWORD> # SSH password
 
 storage:
-  type: WASB # Default is WASB, we can use ADLS_GEN2 as well
-  endpoint: <YOUR_STORAGE_ACCOUNT>.blob.core.windows.net or <YOUR_STORAGE_ACCOUNT>.dfs.core.windows.net # Storage account name 
-  key: YOUR_STORAGE_KEY # Storage key for WASB, not required for ADLS_GEN2
-  resourceGroup: <RESOUCE_GROUP> # Resource group where ADLS Gen2 exist
-  managedIdentityName: <IDENTITY_NAME> # Managed Identity Name
-  mangedIdentityResourceGroup: <IDENTITY_RESOURCE_GREOUP> # Resource Group name where the Managed Identity exist
-
+  type: WASB or ADLS_GEN2 # Default is WASB, we can use ADLS_GEN2 as well
```

Comment:
Just keep WASB as value and mention ADLS_GEN2 in comment or else it is misleading.

Reply:
Done.